playbooks/responses/stale-pr.md

```diff
@@ -6,14 +6,8 @@ action:
 
 Thank you for your contribution to Electron!
 
-We've noticed there hasn't been any activity from the author in some time, and
-since the pull request was opened, there have been many changes to the codebase.
-As a result, your pull request may no longer be applicable or could conflict
-with more recent updates.
+We've noticed there hasn't been any activity from the author in some time, and since the pull request was opened, there have been many changes to the codebase. As a result, your pull request may no longer be applicable or could conflict with more recent updates.
```

Comment:
```suggestion
We've noticed there hasn't been any activity on the pull request in some time, and there have been many changes to the codebase since it was opened. As a result, your pull request may no longer be applicable or could conflict with more recent updates.
```

I edited out the one instance of "the author" because it sounds weird when we're addressing them directly ("you") for the first of the paragraph.